lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -12,6 +12,7 @@
 """
 import subprocess
 import ipaddress
```

Comment:
Do we still need `import ipaddress` if we are not using it anywhere?

Reply:
it is still used after changes.